plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/checkpoint/CheckpointWriter.java

```diff
@@ -161,7 +161,7 @@ public void write(CheckpointEntries entries, TrinoOutputFile outputFile)
         }
         List<DeltaLakeColumnHandle> partitionColumns = extractPartitionColumns(entries.metadataEntry(), entries.protocolEntry(), typeManager);
         List<RowType.Field> partitionValuesParsedFieldTypes = partitionColumns.stream()
-                .map(column -> RowType.field(column.columnName(), column.type()))
+                .map(column -> RowType.field(column.basePhysicalColumnName(), column.type()))
```

Comment:
Looks good now

```
"PartitionValues": {
        "col-6d32b73c-d46b-47f3-aeee-b4ce2231c81f": "30"
      },
      "PartitionValues_parsed": {
        "Col456d32b73c45d46b4547f345aeee45b4ce2231c81f": 30
      }
```

Reply:
Note the missing dashes.

Reply:
dashed should be kept as far as I confirmed Delta Lake protocol and checkpoints Spark generated locally.

https://github.com/delta-io/delta/blob/master/PROTOCOL.md#checkpoint-schema
> 
```
|    |-- partitionValues_parsed: struct
|    |    |-- col-798f4abc-c63f-444c-9a04-e2cf1ecba115: date
|    |    |-- col-19034dc3-8e3d-4156-82fc-8e05533c088e: string
```